<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>898</width>
      <height>999</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>630</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Overall</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>View Grade</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="thisTerm">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>801</width>
       <height>331</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Course ID</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Course Name</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Credits</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Grades</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="printButton">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>860</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Print Grade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="temp">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Temporary</string>
     </property>
    </widget>
    <widget class="QPushButton" name="courseButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Courses</string>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>901</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="status_3">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>30</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OffLine</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="barLabel_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>G-Man Systems</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="profileButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gradeButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Grades</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="overall">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>650</y>
       <width>801</width>
       <height>161</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>GPA</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>GPS</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CRS</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
